HW_7/Exercise_22.ts

```diff
@@ -0,0 +1,69 @@
+//declaring type of recursive array, that can contain either array or any other type
+//type RecursiveArray<T> = T | RecursiveArray<T>[]; 
+
+//declaring new type, that can be any type of list
+type complexType = string|number|boolean|object|null|undefined|symbol|void;
+
+
+/**
+* main function
+* @param numbers (array to be flattened)
+* @returns Array
+*/
+const getFlattenedArray = (numbers: complexType|Iterable<complexType>) => {
```

Comment:
numbers - it's array -> numbers: Number[]

in your case it's elements or something like this -> `elements: Array<Element>` or `elements: Element[]`
/